### PR TITLE
Introduce organizations model

### PR DESCRIPTION
**Pull request by skelz0r**
First step of https://linear.app/pole-api/issue/DAT-137/stockage-des-informations-de-linsee

This pull request introduces proper model for organizations, and populate this table each time we reconcile a user from MonComptePro.

It does not solve the issue above, and is does not change the actual modelisation of legacy organizations management within the app. The main goal here is to ship the model/table, beginning to populate the database, and keep review smalls (ship it!)

Next steps:

1. Create proper relations between models
2. Remove all synchrone calls on INSEE
3. Enhance UX in case of old data from INSEE (for the validator, no need to check INSEE on submits)

### CODE REVIEW

**Comment by skelz0r** [comment]
Side noted: cypress is happy, so the code is working with real life data.

File: backend/app/validators/siret_validator.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  def valid_luhn_checksum?(value)
+    (luhn_checksum(value) % 10).zero?
+  end
+
+  def luhn_checksum(value)
+    accum = 0
+    value.reverse.each_char.map(&:to_i).each_with_index do |digit, index|
+      t = index.even? ? digit : digit * 2
+      t -= 9 if t >= 10
```
Comment (Un3x) [inline backend/app/validators/siret_validator.rb:34]: Hat trick. Nice one

**Review by Un3x — APPROVED** [review]

**Comment by skelz0r** [comment]
I think I will drop the `mon_compte_pro_id` column: siret and this id are bijective according to MCP, so we can just use the siret as an external id key. It will simplify things as well.

**Comment by skelz0r** [comment]
> I think I will drop the `mon_compte_pro_id` column: siret and this id are bijective according to MCP, so we can just use the siret as an external id key. It will simplify things as well.

It was easy, one validated review I merge

**Pull request merged**